server/public/js/userProducts.js

```diff
@@ -37,82 +37,79 @@
-        // for (const name in event.target) {
-        //     console.log(`name`, name);
-        //     formData.append(name, data[name]);
-        // }
+  $(document).bind("click", function (event) {
+    var target = $(event.target);
+    if (target.is(".editProduct")) {
+      console.log("event", event);
+      const productId = event.target.id;
+      console.log(`productId`, productId);
+      let productDetailsRequestConfig = {
+        method: "GET",
+        url: `/products/${productId}/details`,
+        contentType: "application/json",
+      };
+      $.ajax(productDetailsRequestConfig).then(function (response) {
+        console.log(`response for get`, response);
+        $("#updateProductForm").attr(
+          "action",
+          `/products//update/${productId}`
+        );
+        $("#productName").val(response.name);
+        $("#productDescription").val(response.description);
+        $("#productUrl").val(response.site);
+        $("#developer").val(response.developer);
+        $("#developer").val(response.developer);
+        $("#productTag").val(response.tags.toString());
+        $("#productfile").file(`/public/images/upload/${response.logo}`);
+      });
+    }
+  });
+
+  document.forms["updateProductForm"].addEventListener("submit", (event) => {
+    event.preventDefault();
+    const formData = new FormData($("#updateProductForm")[0]);
+    console.log("formData :>> ", formData);
+    // console.log(`event.target`, event.target);
+    // for (const name in event.target) {
+    //     console.log(`name`, name);
+    //     formData.append(name, data[name]);
+    // }
 
-        fetch(event.target.action, {
-            method: "POST",
-            body: formData,
-        })
-            .then((resp) => {
-                return resp.json(); // or resp.text() or whatever the server sends
-            })
-            .then((body) => {
-                console.log(body.error);
-                if (body.error) {
-                    document.querySelector(".errorCreateProduct").innerHTML =
-                        body.error;
-                } else {
-                    console.log(body);
-                }
-            })
-            .catch((error) => {
-                console.log(`error`, error);
-            });
-    });
+    fetch(event.target.action, {
+      method: "POST",
+      body: formData,
+    })
+      .then((resp) => {
+        return resp.json(); // or resp.text() or whatever the server sends
+      })
+      .then((body) => {
+        console.log(body.error);
+        if (body.error) {
+          document.querySelector(".errorCreateProduct").innerHTML = body.error;
+        } else {
+          console.log(body);
+        }
+      })
+      .catch((error) => {
+        console.log(`error`, error);
+      });
+  });
 
-    $("#close").click(function () {
-        $("#exampleModalCenter").modal("hide");
-    });
+  $("#close").click(function () {
+    $("#exampleModalCenter").modal("hide");
+  });
```

Comment:
I dont see much difference in the tabs, I am hoping these are spacing issues from prettier

Reply:
yes spacing issues in this case specifically